### PR TITLE
chore: update wgpu

### PR DESCRIPTION
**Pull request by crowlKats**

### CODE REVIEW

File: ext/webgpu/01_webgpu.js [hunk 1]
```diff
@@ -923,79 +957,51 @@ class InnerGPUDevice {
...
-    const oomFilteredPromise = PromisePrototypeCatch(operation, (err) => {
-      if (ObjectPrototypeIsPrototypeOf(GPUOutOfMemoryErrorPrototype, err)) {
-        return PromiseReject(err);
-      }
-      return PromiseResolve();
-    });
-    if (oomScope) {
-      ArrayPrototypePush(oomScope.operations, oomFilteredPromise);
+    if (scope) {
+      ArrayPrototypePush(constructedError);
```
Comment (0f-0b) [inline ext/webgpu/01_webgpu.js:997]:
```suggestion
      ArrayPrototypePush(scope.errors, constructedError);
```

File: Cargo.lock [hunk 2]
```diff
@@ -3621,6 +3635,12 @@ version = "1.0.11"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "49f1f14873335454500d59611f1cf4a4b0f786f9ac11f4312a78e4cf2566695b"
 
+[[package]]
+name = "jni-sys"
+version = "0.3.0"
```
Comment (littledivy) [inline Cargo.lock:3640]: Hopefully an android-only dependency?

**Review by littledivy — APPROVED** [review]
LGTM!

File: Cargo.lock (hunk 2 above)
Comment (crowlKats) [reply Cargo.lock:3640]: yep, https://github.com/gfx-rs/wgpu/blob/8084eb6c8f31dce4097c000c430afa4f755d7ac0/wgpu-hal/Cargo.toml#L177

**Pull request merged**